testing/postgresql-pglogical/APKBUILD

```diff
@@ -11,9 +11,14 @@ arch="all"
 license="custom"
 depends="postgresql>=$_pgver.0"
 makedepends="libedit-dev libxml2-dev postgresql-dev>=$_pgver.0 zlib-dev"
-source="http://packages.2ndquadrant.com/pglogical/tarballs/$_pkgname-$pkgver.tar.bz2"
+source="http://packages.2ndquadrant.com/pglogical/tarballs/$_pkgname-$pkgver.tar.bz2
+        001-fix-stdin-handling.patch"
 builddir="$srcdir/$_pkgname-$pkgver"
 
+prepare() {
+  default_prepare || return 1
```

Comment:
This is redundant, `default_prepare` is implicitly called when you don't define custom `prepare` function.